frontend/model/contracts/shared/functions.js

```diff
@@ -225,13 +225,13 @@ export function swapMentionIDForDisplayname (
   }
 ): string {
   const {
-    ourContactProfilesById,
     getChatroomNameById,
     usernameFromID,
     userDisplayNameFromID
   } = sbp('state/vuex/getters')
+  const { reverseNamespaceLookups } = sbp('state/vuex/state')
   const possibleMentions = [
-    ...Object.keys(ourContactProfilesById).map(u => makeMentionFromUserID(u).me).filter(v => !!v),
+    ...Object.keys(reverseNamespaceLookups).map(u => makeMentionFromUserID(u).me).filter(v => !!v),
```

Comment:
I actually don't know the exact context that this `reverseNamespaceLookups` field was created for but apparently this field contains user-ids for both current members and members who have been kicked out. So making an update here.

Reply:
@SebinSong this was introduced in this PR: https://github.com/okTurtles/group-income/pull/2294

Nice job realizing it would be useful for this issue!